stock_transport_multi_address/model/procurement.py

```diff
@@ -69,3 +69,8 @@ def _prepare_orderpoint_procurement(self, orderpoint, product_qty):
         res = _super._prepare_orderpoint_procurement(orderpoint, product_qty)
         res.update({'partner_dest_id': orderpoint.warehouse_id.partner_id.id})
         return res
+
+    @api.model
+    def _run_move_create(self, procurement):
+        move_dict = super(ProcurementOrder, self)._run_move_create(procurement)
```

Comment:
Good catch. It could be an artefact of git rebase. I'll remove it and check the build is still green.